Sources/StreamChatUI/ChatMessageList/ChatMessageListView+DiffKit.swift

```diff
@@ -22,7 +22,8 @@ extension ChatMessageListView {
         CATransaction.setCompletionBlock(completion)
         reload(
             using: changeset,
-            with: animation()
+            with: animation(),
+            reconfigure: { _ in false }
```

Comment:
This is a table view and will need more work to efficiently using reconfigure (we have some reuse identifier logic for smart reusing so this needs to take this into an account). Needs to be a separate PR.